src/map/utils/mobutils.cpp

```diff
@@ -928,8 +928,10 @@ void SetupNMMob(CMobEntity* PMob)
 
 void SetupMaat(CMobEntity* PMob)
 {
-    // Should swing normally when not a mnk
-    PMob->m_Weapons[SLOT_MAIN]->setDelay((240*1000)/60);
+    // I arrived at this attack speed value by comparing his attacks to
+    // videos of retail Maat.  I believe it's base attack rate of 480
+    // minus six levels of martial arts (-20 each)
+    PMob->m_Weapons[SLOT_MAIN]->setDelay((360*1000)/60);
```

Comment:
The idea behind it was whoever did Maat before thought his delay should be lower when he wasn't both-hand punching you.

This whole on the fly set Maats job etc is non retail. If you look in retail each jobs Maat fight spawns a different Maat (they have diff IDs, and you can see multiples of him in the clients dat file via pol utils). But lets get this merged so it goes back to working the way it did before and worry about anything else later.